utils/index.d.ts

```diff
@@ -360,6 +360,7 @@ export interface IAzExtParentTreeItem extends IAzExtTreeItem {
  * NOTE: *Impl methods are not meant to be called directly - just implemented.
  */
 export declare abstract class AzExtTreeItem implements IAzExtTreeItem {
+    public readonly _isAzExtTreeItem = true;
```

Comment:
I don't think this should be public. Instead we should declare and provide an `isAzExtTreeItem(maybeTreeItem: unknown): maybeTreeItem is AzExtTreeItem`.

Reply:
Removed it from the declaration, and added `isAzExtTreeItem(maybeTreeItem: unknown): maybeTreeItem is AzExtTreeItem`.